client/src/app/app.component.ts

```diff
@@ -107,8 +107,7 @@ export class AppComponent {
         appRef.isStable
             .pipe(
                 // take only the stable state
-                filter(s => s),
-                take(1)
+                first(stable => stable)
```

Comment:
Why did you remove the `take(1)`. This event should only be emitted once.

Reply:
As of the description from rxjs:

> first
> Emits only the first value. Or emits only the first value that passes some test.

I think, `first` will unify filter and take. ;)